### DISCUSSION TITLE
思维导图 |《创业维艰》读书笔记 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#116] [Announcements]
# 思维导图 |《创业维艰》读书笔记 · 技术助力业务

本文分享了《创业维艰》这本书的读书笔记以及对创业的思考

https://venerable-tiramisu-d9881e.netlify.app/self/the-hard-thing/

**Comment by wyx2014** [comment]
hello world
